Source/WebKit/UIProcess/API/glib/WebKitUserContentManager.cpp

```diff
@@ -224,12 +247,64 @@ void webkit_user_content_manager_remove_all_scripts(WebKitUserContentManager* ma
     manager->priv->userContentController->removeAllUserScripts();
 }
 
+/**
+ * webkit_user_script_message_get_value:
+ * @message: A #WebKitUserScriptMessage
+ *
+ * Return the message sent in by user.
+ *
+ * Returns: (transfer none): The #JSCValue sent by JavaScript code to the
+ * script message handler.
+ *
+ * Since: 2.38
+ */
+JSCValue* webkit_user_script_message_get_value(WebKitUserScriptMessage* userScriptMessage)
+{
+    if (!userScriptMessage->message)
+        return jsc_value_new_undefined(SharedJavaScriptContext::singleton().getOrCreateContext());
```

Comment:
The function is annotated with `Return: (transfer none)`, which indicates that the caller does _not_ need to take care of doing an unref on the returned value. But this returns a new value, with a reference count of `1`, and if the caller never unrefs it, the value will be leaked.

We should make sure inside the `_WebKitUserScriptMessage()` constructor that `userScriptMessage->message` will always be a valid object instead.